### ISSUE TITLE
crash when opening small mp4 movie

### ISSUE THREAD

**Issue by greenorca** [nextcloud/android#7435]
### Steps to reproduce
1. Add attached video file to shared nextcloud folder
2. Open the video in nextcloud app
3. 

### Expected behaviour
- video should be played

### Actual behaviour
- app crashes
- other video players can actually play the downloaded video file 

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version: see log

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version: 

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
************ CAUSE OF ERROR ************

android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground()
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1821)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6626)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:811)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: motorola
Device: potter_n
Model: Moto G (5) Plus
Id: OPS28.85-17-6-2
Product: potter_n

************ FIRMWARE ************
SDK: 27
Release: 8.1.0
Incremental: 77e7

```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by zz-1b** [comment]
I get the same crashes, also when playing videos or downloading photos. Seems to happen more frequently when the network connection is slow / via VPN tunnel

[CAUSE OF ERROR.txt](https://github.com/nextcloud/android/files/5579317/CAUSE.OF.ERROR.txt)

**Comment by tobiasKaminsky** [comment]
Will be fixed via #7340
